resources/psi-ms-PARSED.xml

```diff
@@ -211,6 +214,7 @@
     <instrument id="MS:1002877" name="Q Exactive HF-X" vendor="Thermo Scientific" description="&quot;Thermo Scientific Q Exactive HF-X Hybrid Quadrupole Orbitrap MS.&quot; [PSI:PI]"/>
     <instrument id="MS:1000644" name="TSQ Quantum Access" vendor="Thermo Scientific" description="Thermo Scientific TSQ Quantum Access MS."/>
     <instrument id="MS:1000645" name="Element XR" vendor="Thermo Scientific" description="Thermo Scientific Element XR HR-ICP-MS."/>
+    <instrument id="MS:1003028" name="Orbitrap Exploris 480" vendor="Thermo Scientific" description="&quot;Thermo Scientific Orbitrap Exploris 480 Quadrupole Orbitrap MS.&quot; [PSI:PI]"/>
```

Comment:
Same thing - double quote

Reply:
Looks like the OBO parser was returning double quotes that were not getting trimmed off in all cases.  They were removed with the last commit 36788a703d3a152ff40e5d7c5272976e035d6f7f.